--- a/package/libs/ustream-ssl/Makefile
+++ b/package/libs/ustream-ssl/Makefile
@@ -1,15 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ustream-ssl
-PKG_SOURCE_DATE:=2025-10-03
+PKG_SOURCE_DATE:=2024-07-28
 PKG_SOURCE_VERSION:=5a81c108d20e24724ed847cc4be033f2a74e6635
 PKG_RELEASE:=1
 
 PKG_LICENSE:=ISC
 PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://git.openwrt.org/project/ustream-ssl.git
+PKG_SOURCE_URL:=https://github.com/openwrt/ustream-ssl.git
 PKG_MIRROR_HASH:=e251189ed315f22ab63dc6f17b03178676e10c21fff0cdd863b294a3c51a1b5b
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>